src/components/implementation/no_interface/scheddev/fprr/Makefile

```diff
@@ -0,0 +1,9 @@
+#C_OBJS=unit_schedlib.o
```

Comment:
I see.  You're trying to create a directory hierarchy in scheddev for the different scheduling modules.

This seems to be a "one-off" solution for the scheddev component, though.  If you're going so far as to break apart different modules, should we just do that in the `components/lib/` directory so that we can include them in each component separately?

Still might be missing something big here, so I'll read on.